slack.py

```diff
@@ -21,6 +21,7 @@
 BOT_EMOJI = CONFIG['slack']['bot']['emoji'].get()
 COST_ALERT_THRESHOLD = CONFIG['slack']['cost_alert_threshold'].get(float)
 COST_ALERT_EMOJI = CONFIG['slack']['cost_alert_emoji'].get()
+COST_MIN_TO_NOTIFY = CONFIG['chat']['cost_min_to_notify'].get(float)
```

Comment:
It's CONFIG['slack'].

Reply:
these changes are made in commit 9f13dbc